pmm/pmm2-package-testing.groovy

```diff
@@ -98,7 +98,7 @@ pipeline {
             description: 'Type of Tests?',
             name: 'TESTS')
         choice(
-            choices: ['testing', 'experimental', 'main'],
+            choices: ['testing', 'experimental', 'main', 'tools_main', 'pmm2_client_main'],
```

Comment:
I honestly don't like the name of the repos here. It seems a dash would be better: `tools-main`, `pmm2-client-main`.
But I don't have a strong opinion here;

Reply:
okay, I think I agree with your point. Let me make changes.